Mathlib/GroupTheory/QuotientGroup/Defs.lean

```diff
@@ -91,6 +91,10 @@ theorem eq_one_iff {N : Subgroup G} [N.Normal] (x : G) : (x : G ⧸ N) = 1 ↔ x
   refine QuotientGroup.eq.trans ?_
   rw [mul_one, Subgroup.inv_mem_iff]
 
+@[to_additive (attr := simp)]
+theorem range_mk' : (QuotientGroup.mk' N).range = ⊤ :=
```

Comment:
I agree this does not need a docstring explaining why this needs a `'` at the end. So you can add the name to `scripts/nolint_prime_decls.txt` to avoid the linter warning.

Reply:
Perhaps even with a comment that this is a false positive in the linter.

Reply:
I've left a doc-string for `range_mk'` in the `Defs`, and used `set_option` in the section for the correspondence theorem in `Basics`.